apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/Familieforhold.tsx

```diff
@@ -24,7 +24,11 @@ export const Familieforhold = ({ behandling, personopplysninger }: PropsFamilief
         <Heading spacing size="medium" level="2" as="div">
           Familieforhold
           {featureAktiv && personopplysninger && (
-            <RedigerFamilieforhold behandling={behandling} personopplysninger={personopplysninger} />
+            <RedigerFamilieforhold
+              behandling={behandling}
+              personopplysninger={personopplysninger}
+              sakType={behandling.sakType}
```

Comment:
Er det behov for å ta inn behandling.sakType når du allerede tar inn behandling?

Reply:
Det er det ikke, good catch